specs/waku/v2/waku-store.md

```diff
@@ -2,36 +2,88 @@
 title: Waku
 version: 2.0.0-beta1
 status: Draft
-authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>, Sanaz Taheri <sanaz@status.im>
 ---
 
 # Table of Contents
 
 - [Abstract](#abstract)
+  * [Waku Store Protocol](#Waku-Store-Protocol)
+  * [Pagination](#Pagination)
 - [Wire Specification](#wire-specification)
   * [Protobuf](#protobuf)
 - [Changelog](#changelog)
 - [Copyright](#copyright)
 
 # Abstract
 
+## Waku Store Protocol 
```

Comment:
```suggestion
```
I think this still makes sense as an abstract, where pagination is one aspect of the store protocol. Otherwise they sound disjointed to me